utils/linker/crc32.js

```diff
@@ -10,12 +10,12 @@
  * @returns array of CRC table
  */
 
-const makeCRCTable = () => {
+const makeCRCTable = function () {
   const crcTable = [];
   let c;
-  for (let n = 0; n < 256; n += 1) {
```

Comment:
Why change the syntax? Both do the same thing.

Reply:
`eslint` recommendation.

Reply:
Okay